### DISCUSSION TITLE
codec not available: 'blosc2'

### DISCUSSION THREAD

**Discussion by lakatadam21** [openclimatefix/community#8] [Help] [answered]
Hi,

I would like to create a series of day-ahead forecasted solar generation to be included in bidding strategy simulations for 2020-2021. I tried to use the zarr package posted for the OCF Climate Hack competition: https://huggingface.co/datasets/climatehackai/climatehackai-2023/tree/main/weather/2020 but I get an error of "codec not available: 'blosc2'", despite installing the necessary packages. 

Is there an alternative dataset I could use that doesn't include all the weather forecast variables only a single GHI forecast variable for the PV sites included in the PV-UK dataset for example?

Thanks,
Adam

**Comment by peterdudfield** [comment, marked as answer]
Ah, that error can be solved by install [ocf_blosc2](https://pypi.org/project/ocf-blosc2/) and `import ocf_blosc2` before running our code

**Reply by lakatadam21** [reply]
Thank you, that fixed it :)

**Reply by peterdudfield** [reply]
Super, that's great news
